python/cuml/metrics/hinge_loss.pyx

```diff
@@ -0,0 +1,120 @@
+#
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import cupy as cp
+from cuml.preprocessing import LabelEncoder, LabelBinarizer
+import cudf
+
+
+def cython_hinge_loss(y_true, pred_decision, labels=None, sample_weights=None):
+    """
+    Calculates non-regularized hinge loss. Adapted from scikit-learn hinge loss
+
+    Parameters
+    ----------
+    y_true: cuDF Series or cuPy array of shape (n_samples,)
+            True labels, consisting of labels for the classes.
+            In binary classification, the positive label must be
+            greater than negative class
```

Comment:
```suggestion
    True labels, consisting of labels for the classes.
    In binary classification, the positive label must be
    greater than negative class
```
I wouldnt align them like this. There is only 80 characters. Need all the room you can get.